src/table.ts

```diff
@@ -77,205 +77,202 @@ export class Table<
   /**
    * @see https://docs.aws.amazon.com/appsync/latest/devguide/resolver-mapping-template-reference-dynamodb.html#aws-appsync-resolver-mapping-template-reference-dynamodb-getitem
    */
-  // @ts-ignore
-  public getItem<
-    Key extends TableKey<
-      Item,
-      PartitionKey,
-      RangeKey,
-      JsonFormat.AttributeValue
-    >
-  >(input: {
-    key: Key;
-    consistentRead?: boolean;
-  }): Narrow<Item, AttributeKeyToObject<Key>, JsonFormat.Document>;
 
-  public getItem(call: CallExpr, vtl: CallContext): any {
-    assertIsVTLContext(vtl, "getItem");
-
-    const input = vtl.eval(
-      assertNodeKind<ObjectLiteralExpr>(
-        call.getArgument("input")?.expr,
-        "ObjectLiteralExpr"
-      )
-    );
-    const request = vtl.var(
-      `{"operation": "GetItem", "version": "2018-05-29"}`
-    );
-    vtl.qr(`${request}.put('key', ${input}.get('key'))`);
-    addIfDefined(vtl, input, request, "consistentRead");
+  public getItem = makeIntegration<
```

Comment:
Same question around documentation and ide ergonomics. Let's make sure these convenience methods don't hurt the DX for a user.

Reply:
With `const`

![image](https://user-images.githubusercontent.com/289213/165332832-23c863e3-6667-4321-ad22-83a81c456a54.png)

![image](https://user-images.githubusercontent.com/289213/165332075-912f4dbc-559a-4bb7-909b-1b94c60396ae.png)

With function

![image](https://user-images.githubusercontent.com/289213/165332410-5d6b27fa-06e9-48d0-bd6c-a968aad4fcfd.png)

![image](https://user-images.githubusercontent.com/289213/165332363-59d8d2d5-3426-4fea-97d4-64e7190d162d.png)